system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.cpp

```diff
@@ -85,4 +85,10 @@ std::string LinuxCpuMeasurementNode::GetMetricName() const
   return kMeasurementType;
 }
 
+const std::string & LinuxCpuMeasurementNode::GetMetricUnit() const
+{
+  static const std::string unit_name{"percent"};
```

Comment:
Since the same string is used elsewhere it could be a constant.

Reply:
Done.